src/engraving/rw/read206/read206.cpp

```diff
@@ -3022,32 +3022,21 @@ static void readMeasure206(Measure* m, int staffIdx, XmlReader& e, ReadContext&
 
 static void readBox(Box* b, XmlReader& e, ReadContext& ctx)
 {
-    b->setLeftMargin(0.0);
-    b->setRightMargin(0.0);
-    b->setTopMargin(0.0);
-    b->setBottomMargin(0.0);
-    b->setTopGap(Millimetre(0.0));
-    b->setBottomGap(Millimetre(0.0));
-    b->setAutoSizeEnabled(false);
-    b->setPropertyFlags(Pid::TOP_GAP, PropertyFlags::UNSTYLED);
-    b->setPropertyFlags(Pid::BOTTOM_GAP, PropertyFlags::UNSTYLED);
```

Comment:
Actually removing the above two lines fixed that styled issue with Mu2 scores that started their live in Mu1 already.
Only they still show their bottom gap 1.0sp larger than in Mu2.
Mu1 default was 1.0, Mu2 default was 0.0 (so the import marked the Mu1 1.0 as unstyled), Mu3 and Mu4 default is 7.0 and when reading a Mu2 score the current (unstyled) value and the current default get added, at least for the top frame, see a few lines down, in `readStaffContent206()`